src/fhi/blocks/_fhi-error-page.block.scss

```diff
@@ -0,0 +1,20 @@
+.fhi-error-page {
+    &__heading {
+        @extend .h2;
+        margin-bottom: $fhi-core-space-3;
+
+        &__status-code {
```

Comment:
No need to nest this one into heading

Reply:
Sure? I'd say it is dependant on the .fhi-error-page__heading

Reply:
Ah, I see the BEM info documentation. Got it.